src/main/java/dinkplugin/notifiers/ChatNotifier.java

```diff
@@ -158,6 +161,10 @@ private ClanTitle getClanTitle(@NotNull ChatMessageType type, @Nullable String n
         return settings.titleForRank(member.getRank());
     }
 
+    private static String getSender(ChatNotificationType type, String source) {
+        return source == null || source.isEmpty() || type == COMMAND || type == RUNELITE ? "[" + type + "]" : source;
```

Comment:
Could we un-ternary this?
```suggestion
		if (source == null || source.isEmpty() || type == COMMAND || type == RUNELITE) {
			return "[" + type + "]";
		}
		return source;
```